google-analytics-admin/pom.xml

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>com.google.analytics</groupId>
+  <groupId>com.google.cloud</groupId>
```

Comment:
do we publish this artifact? If so, this shouldn't be changing.

Reply:
Reverted the group id to match the published artifact.